portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -12,7 +15,18 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+
+    ArrayList<String> greetings = new ArrayList<>();
+    greetings.add("hello");
+    greetings.add("hi");
+    greetings.add("whats up");
+
+    response.setContentType("application/json;");
+    response.getWriter().println(convertToJsonUsingGson(greetings));
+  }
+  private String convertToJsonUsingGson(ArrayList a) {
```

Comment:
Can you make the ArrayList variable have a more descriptive name? It's hard to keep track of what code is doing when you have to go back and search what type a variable is every time it's referenced.